### ISSUE TITLE
[Bug] [Android] Our recordings from Jetpack Compose are blank

### ISSUE THREAD

**Issue by hugoangeles0810** [microsoft/clarity-apps#77]
**Describe the Bug**
All of our recordings are blank on Clarity Dashboard, we don't see any Clarity error log on Logcat

![Image](https://github.com/user-attachments/assets/36342ffb-13cb-4285-bf3f-482a03f4d691)
 

**Setup**
Tried in  `Application.onCreate()` and `MainActivity.onCreate()`
```kotlin
val config = ClarityConfig(BuildConfig.CLARITY_PROJECT_ID)
Clarity.initialize(this, config)
```

**Platform/Framework Info**
Device: Samsung Galaxy S23 Ultra, Xiaomi Redmi Note.
OS: Android, versions 10 and 14
Application Framework Android with Jetpack Compose
Clarity Version tested: 2.5.0 and 2.5.2
Compose Version: 1.6.0

**Screenshots**
![Image](https://github.com/user-attachments/assets/47ae1ca1-9d26-4cdf-9589-fd9d367a95de)

![Image](https://github.com/user-attachments/assets/ad61c98a-a865-48dd-8c6e-ed1ad4463567)

**Comment by hugoangeles0810** [comment]
Hi @ibradwan,

Yes, I have used clarity-compose 2.5.0 and also 2.5.2. We are getting blank screens with both versions.

**Comment by SaadeldinBadr** [comment]
Hello @hugoangeles0810,

please enable logs, by modifying config with:

```
        val config = ClarityConfig(
            BuildConfig.CLARITY_PROJECT_ID,
            logLevel = LogLevel.Verbose,
        )
```

share logs with us and most probably we will see what causes the problem from Clarity side.

**Comment by hugoangeles0810** [comment]
Hello @SaadeldinBadr 

I've tried enabling logs and we can see this error on LogCat

```
2024-12-03 16:25:48.186 25409-25409 Clarity                 app...o_nicolas_salvatto.Sordos.dev  E[SourceFile::a] Exception in captureAndroidComposeView: java.lang.NullPointerException
2024-12-03 16:25:48.187 25409-25409 Clarity                 app...o_nicolas_salvatto.Sordos.dev  E  [SourceFile::a] null
2024-12-03 16:25:48.188 25409-25409 Clarity                 app...o_nicolas_salvatto.Sordos.dev  E  [SourceFile::a] java.lang.NullPointerException
                                                                                                    	at com.microsoft.clarity.e.b.a(Unknown Source:33)
                                                                                                    	at com.microsoft.clarity.e.b.a(SourceFile:6)
                                                                                                    	at com.microsoft.clarity.e.d.a(SourceFile:84)
                                                                                                    	at com.microsoft.clarity.e.d.a(SourceFile:84)
                                                                                                    	at com.microsoft.clarity.e.d.a(SourceFile:84)
                                                                                                    	at com.microsoft.clarity.e.d.a(SourceFile:84)
                                                                                                    	at com.microsoft.clarity.e.d.a(SourceFile:84)
                                                                                                    	at com.microsoft.clarity.e.d.a(SourceFile:84)
                                                                                                    	at com.microsoft.clarity.e.d.a(SourceFile:84)
                                                                                                    	at com.microsoft.clarity.e.d$d.invoke(SourceFile:3)
                                                                                                    	at com.microsoft.clarity.e.d.a(SourceFile:20)
                                                                                                    	at com.microsoft.clarity.f.j.invoke(SourceFile:3)
                                                                                                    	at com.microsoft.clarity.n.k.a(Unknown Source:22)
                                                                                                    	at com.microsoft.clarity.f.k$a.invoke(SourceFile:12)
                                                                                                    	at com.microsoft.clarity.n.d.a(Unknown Source:5)
                                                                                                    	at com.microsoft.clarity.n.c.invoke(SourceFile:1)
                                                                                                    	at com.microsoft.clarity.n.k.a(Unknown Source:22)
                                                                                                    	at com.microsoft.clarity.n.d.a(SourceFile:5)
                                                                                                    	at com.microsoft.clarity.f.k.run(SourceFile:2)
                                                                                                    	at android.os.Handler.handleCallback(Handler.java:958)
                                                                                                    	at android.os.Handler.dispatchMessage(Handler.java:99)
                                                                                                    	at android.os.Looper.loopOnce(Looper.java:230)
                                                                                                    	at android.os.Looper.loop(Looper.java:319)
                                                                                                    	at android.app.ActivityThread.main(ActivityThread.java:9063)
                                                                                                    	at java.lang.reflect.Method.invoke(Native Method)
                                                                                                    	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:588)
                                                                                                    	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1103)
```

**Comment by SaadeldinBadr** [comment]
@hugoangeles0810 This exception is caused due to using an unsupported Compose version. Even if you specify Compose 1.6.0 in your dependencies, it might have been upgraded to a higher version due to another dependency (compose navigation for example). Could you provide a list of all Compose-related dependencies you’re using, including their versions?

**Comment by hugoangeles0810** [comment]
You were right @SaadeldinBadr!

After a completed dependencies scan, we could find a dependency conflict that upgraded our expected compose version from 1.6.0 to 1.7.2.

Now we can see the recordings, thanks for your support guys 🥇

**Comment by SaadeldinBadr** [comment]
No problem, thanks for reaching out! 🙂

**Comment by SaadeldinBadr** [comment]
Hello @hugoangeles0810,
We've just released `clarity-compose:3.1.0` , which supports jetpack compose `1.7.*`
so feel free to upgrade your compose and clarity versions.

**Comment by vishal-bachatt** [comment]
getting black screen in my kmp android project
clarity compose 3.8.3
compose 1.11.1
@hugoangeles0810 can you help here
